web/src/app/administration/components/classification-overview/classification-overview.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable, Subject } from 'rxjs';
+import { Select, Store } from '@ngxs/store';
+import { takeUntil } from 'rxjs/operators';
+import { ClassificationSelectors } from '../../../shared/store/classification-store/classification.selectors';
+import { ClassificationDefinition } from '../../../shared/models/classification-definition';
+import { ACTION } from '../../../shared/models/action';
+import { SelectClassification,
+  SetActiveAction } from '../../../shared/store/classification-store/classification.actions';
+
+@Component({
+  selector: 'app-classification-overview',
+  templateUrl: './classification-overview.component.html',
+  styleUrls: ['./classification-overview.component.scss']
+})
+export class ClassificationOverviewComponent implements OnInit, OnDestroy {
+  showDetail = false;
+  @Select(ClassificationSelectors.selectedClassification) selectedClassification$: Observable<ClassificationDefinition>;
```

Comment:
We should rename ClassificationDefinition to Classification. (Can also do this later)